urbit/bridge
# Invite Acceptance Flow

# Preview

https://user-images.githubusercontent.com/16504501/133674423-dc80aeba-e12b-47b2-8f29-5b1cd75ae9cd.mov

# Context

This PR adds an all new Activation flow for new users to activate an Invite generated in Bridge.

# Changes

- New Activation flow built using `@tlon/indigo-react` component library

# Testing

Confirmed with local instance of Urbit binary running `naive/roller` branch:

1. Start the local environment as described in the latter section of `DEVELOPMENT.md`
2. Log in to Bridge, select ~wanzod
3. Click the envelope and generate some invites
4. After the next batch rolls, check the pending invites tab by logging out and in again and clicking the envelope
5. Use one of the links, format it like so, and visit to start the Accept Invite Flow:
`localhost:3000/#pitbep-salleb-nimrum-dinwer-dalnut-dovtux`
6. After a successful transfer of the point, trying to use the code again fails as expected
